src/platform-release.js

```diff
@@ -228,13 +228,15 @@ exports.prepareReleaseBranchCommand = function * () {
                 yield executil.execHelper(executil.ARGS('git checkout -b ' + branchName));
             }
 
+            print(repo.repoName + ': Update JS snapshot version for VERSION "' + version + '" on branch "' + branchName + '".');
```

Comment:
Both changes from this commit (aefb169daa57f3931b08aa8ea63866dcd58117ba) were clobbered by a4110226b5e5ad9f36a1782dd8ca972870c0f0cb and will likely go away.